var/spack/repos/builtin/packages/minimap2/package.py

```diff
@@ -32,3 +52,9 @@ def install_minimap2(self):
         make(*make_arg)
         mkdirp(prefix.bin)
         install("minimap2", prefix.bin)
+        sed = which("sed")
+        if self.spec.satisfies("js_engine=node"):
+            sed("-ie", "1s/k8/node/", "./misc/paftools.js")
+            # sed("-ie", "1s/k8/node/", "./misc/mmphase.js")
+        install("./misc/paftools.js", prefix.bin)
+        # install("./misc/mmphase.js", prefix.bin)
```

Comment:
What are these things that are commented out? Can they be removed?

We shouldn't use sed as it won't exist on Windows, we should use `filter_file` instead.